lib/message_generator.dart

```diff
@@ -27,10 +27,27 @@ class MessageGenerator extends ProtobufContainer {
   /// The name of the Dart class to generate.
   final String classname;
 
-  /// The fully-qualified name of the message type.
+  /// The fully-qualified name of the message (without any leading '.').
+  final String fullName;
+
+  /// The part of the fully qualified name that comes after the package prefix.
+  ///
+  /// For nested messages this will include the names of the parents.
   ///
-  /// (Used as a unique key and in error messages, not in Dart code.)
-  final String fqname;
+  /// For example:
+  /// ```
+  /// package foo;
+  ///
+  /// message Container {
+  ///   message Nested {
+  ///     int32 int32_value = 1;
+  ///   }
+  /// }
+  /// ```
+  /// The nested message will have a `fullName` of 'foo.Container.Nested', and a
+  /// `messageName` of 'Container.Nested'.
```

Comment:
This comment seems in conflict with the google.protobuf.{DescriptorProto.}ExtensionRange below.